lib/public/components/common/table/table.js

```diff
@@ -135,10 +137,10 @@ export const table = (
         Error(`Unhandled type <${typeof data}> of data : ${data ? JSON.stringify(data) : data}`);
     }
 
-    return h(
+    return h(`${horizontalScrollEnabled ? '.scroll-auto' : ''}`, h(
```

Comment:
With the addition of a new `h` element, you are losing the UI delimiter of the table: `shadow-level` which makes it difficult from a visual point to see where the table ends. 

Your changes:
<img width="250" alt="image" src="https://github.com/AliceO2Group/Bookkeeping/assets/9214854/250b092a-a72f-419c-bec1-9bf46e35f0b3">

Current behaviour:
<img width="231" alt="image" src="https://github.com/AliceO2Group/Bookkeeping/assets/9214854/b3bb21de-7721-4f56-b234-2bac2bd14d76">

Reply:
right, fixed